app/assets/stylesheets/zeal-initiative-tracker.webflow.css

```diff
@@ -288,6 +291,9 @@ h5 {
   font-family: Lato, sans-serif;
   font-size: 16px;
   line-height: 1.4;
+  position: relative;
+  z-index: 1;
+
```

Comment:
Please remove empty lines within style block

Reply:
this issue was resolved